script.js

```diff
@@ -0,0 +1,7 @@
+const radius = 5;
+const area = getCircleArea(radius);
+console.log(area); // 78.53981633974483
+
+function getCircleArea(providedNumber) {
```

Comment:
getCircleArea(`providedNumber`) is not the best name as we know we will provide radius... maybe `circleRadius` instead? :)